charts/i2b2/values.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for i2b2.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: i2b2/i2b2-web
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v1.8.0.0002"
```

Comment:
Why not putting this in the `appVersion` then?

Reply:
Done